src/Committee/Election/CandidacyManager.php

```diff
@@ -65,11 +65,20 @@ public function updateCandidature(
         );
     }
 
-    public function getCandidacy(Adherent $adherent, Committee $committee): ?CommitteeCandidacy
-    {
+    public function getCandidacy(
+        Adherent $adherent,
+        Committee $committee,
+        bool $createIfNotExist = false
+    ): ?CommitteeCandidacy {
         $membership = $adherent->getMembershipFor($committee);
 
-        return $membership->getCommitteeCandidacy($committee->getCommitteeElection());
+        $candidacy = $membership->getCommitteeCandidacy($election = $committee->getCommitteeElection());
+
+        if ($createIfNotExist && !$candidacy) {
+            return $this->createCandidacy($election, $adherent->getGender());
```

Comment:
it seems `$election` may be null at this point

Reply:
fixed by checking `$election` value 👍